packages/devtools/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   js: ^0.6.1+1
   meta: ^1.1.0
   path: ^1.6.0
-  pedantic: ^1.5.0
+  pedantic: ^1.4.0
```

Comment:
This downgrade is intentional?

Reply:
yes. 1.5 requires too new a dart:sdk for pub.dartlang.org to handle
resulting in getting a pub score of zero on pub.dartlang.org instead of a score of ~80 for the package health questions.